components/resources/jobExplorer/jobList/JobListComponent.tsx

```diff
@@ -135,18 +150,41 @@ const JobListComponent = () => {
     },
   ];
 
-  const beginnerJobs = jobs.filter((job) => job.title.includes("Front End Developer") || job.title.includes("Back End Developer") || job.title.includes("Data Scientist"));
-  const intermediateJobs = jobs.filter((job) => job.title.includes("Fullstack Developer") || job.title.includes("Mobile Developer") || job.title.includes("Game Developer"));
-  const advancedJobs = jobs.filter((job) => job.title.includes("AI Engineer") || job.title.includes("Cyber Security Analyst") || job.title.includes("DevOps Engineer"));
-  const nonTechnicalJobs = jobs.filter((job) => job.title.includes("UX Designer") || job.title.includes("Product Manager") || job.title.includes("Digital Marketer"));
+  const beginnerJobs = jobs.filter(
+    (job) =>
+      job.title.includes("Front End Developer") ||
+      job.title.includes("Back End Developer") ||
+      job.title.includes("Data Scientist")
+  );
+  const intermediateJobs = jobs.filter(
+    (job) =>
+      job.title.includes("Fullstack Developer") ||
+      job.title.includes("Game Developer") ||
+      job.group === "mobile"
```

Comment:
rather than search for every title, i can easily search for all jobs with mobile as the group